### DISCUSSION TITLE
[ Implemented ]  Update button for sketches

### DISCUSSION THREAD

**Discussion by Intuos5** [hlorus/CAD_Sketcher#138] [Feature Requests]
**Motivation / Existing Problem**
When moving a workplane, the sketch that is associated to a workplane needs to be updated. This is currently done by entering and exiting the sketch. 
https://youtu.be/4Al3yw2klHA?t=353

**Description**
Ideally, there would be an update button next to the pencil button, which would update the sketch in a single click. This saves users from having to enter and exit a number of sketches. 

**Alternative Solutions**
Entering and exiting sketches is possible, but requires more clicks and changes the display of the object as well. Ideally, you can immediately see the changes in the mesh.

**Mockup**
![Reload](https://user-images.githubusercontent.com/63149655/170222036-0ba28edf-d44d-4e59-8a89-957ea212036c.png)

**Comment by hlorus** [comment]
There's now a global "Force Update" button which can be used to reconvert all sketches. Not sure if this should be possible per sketch...
